cssom-1/Overview.bs

```diff
@@ -2034,6 +2034,25 @@ Note: The serialization of an empty CSS declaration block is the empty string.
 Note: The serialization of a non-empty CSS declaration block does not include any surrounding whitespace, i.e., no whitespace appears
 before the first property name and no whitespace appears after the final semicolon delimiter that follows the last property value.
 
+A <a>CSS declaration block</a> has these <a>attribute change steps</a> for its <a for="CSSStyleDeclaration">owner node</a>:
+<ol>
+ <li>Assert: the <a for="CSSStyleDeclaration">readonly flag</a> is unset.
+ <li>If <var>localName</var> is not "<code>style</code>", or <var>namespace</var> is not null, terminate these steps.
+ <li>If <var>value</var> is null, empty the <a for="CSSStyleDeclaration">declarations</a>.
+ <li>Otherwise, let the <a for="CSSStyleDeclaration">declarations</a> be the result of <a>parse a CSS declaration block</a>
+ from a string <var>value</var>.
+</ol>
+
+When a <a>CSS declaration block</a> object is created, then:
+<ol>
+ <li>Let <var>owner node</var> be the <a for="CSSStyleDeclaration">owner node</a>.
```

Comment:
Need to also check if the readonly flag is set. Now this will try to get an attribute when doing `getComputedStyle(elm)` I think.

Reply:
Oh, you're right. The algorithm for `attribute change steps` also need updates.

Reply:
Updated to make this algorithm and `attribute change steps` check `readonly flag`.